src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int valueNumber = 100;
```

Comment:
1. constants should be uppercased
2. keep blank line between constants and fields
3. bad constant name, its totally unclear what does it store
```suggestion
    public static final int BALL_MAX_NUMBER= 100;
    
```